static/downloadables/README.md

```diff
@@ -43,8 +43,14 @@ To run the risk score calculator from the command-line, you should pass the requ
 
 #### Using a VCF with required parameters
 ```bash
-./runPrsCLI.sh -f inputFile.vcf -o outputFile.tsv -r hg19 -c 0.05 -p EUR
+./runPrsCLI.sh -f 'inputFiles_chr*.vcf' -o outputFile.tsv -r hg19 -c 0.05 -p EUR
```

Comment:
Should this line have been changed? It looks identical to the line under the next heading.

Reply:
good catch!